### PR TITLE
Add docs for packaging changes

### PR DESCRIPTION
**Pull request by dedemorton**
Documents new directory structure, settings files, and instructions for running Logstash as a service.

This content will appear in the following location.

settinguporg

### CODE REVIEW

File: docs/static/setting-up-logstash.asciidoc [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+| bin
+  | Binary scripts, including `logstash` to start Logstash
+    and `logstash-plugin` to install plugins
+  | `{extract.path}/bin`
+ d|
+
+| conf
+  | Configuration files, including `logstash.yml` and `jvm.options`
+  | `{extract.path}/config`
+  | `path.conf`
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: This should be `path.settings`

File: docs/static/setting-up-logstash.asciidoc [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+ d| `{extract.path}`- Directory created by unpacking the archive
+  | `path.home`
+
+| bin
+  | Binary scripts, including `logstash` to start Logstash
+    and `logstash-plugin` to install plugins
+  | `{extract.path}/bin`
+ d|
+
+| conf
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: This should be "settings"

File: docs/static/setting-up-logstash.asciidoc [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  | `/usr/share/logstash`
+ d| `path.home`
+
+| bin
+  | Binary scripts including `logstash` to start Logstash
+    and `logstash-plugin` to install plugins
+  | `/usr/share/logstash/bin`
+ d|
+
+| conf
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: This should be "settings"

File: docs/static/setting-up-logstash.asciidoc [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  | Configuration files, including `logstash.yml`, `jvm.options`, and `startup.options`
+  | `/etc/logstash`
+  | `path.conf`
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: This should be `path.settings`

File: docs/static/setting-up-logstash.asciidoc [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+| conf
+  | Logstash pipeline configuration files
+  | `/etc/logstash/conf.d`
+ d|
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: This should be `path.conf`

File: docs/static/setting-up-logstash.asciidoc [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  Contains JVM configuration flags. Specify each flag on a separate line. You can also use this file to set the locale
+  for Logstash.
+*`startup.options` (Linux)*::
+  Contains options used by the `system-install` script in `/usr/share/logstash/bin` to build the appropriate startup
+  script for your system. When you install the Logstash package, the `system-install` script executes at the end of the
+  installation process and uses the settings specified in `startup.options` to set options such as the user, group,
+  service name, and service description. By default, Logstash services are installed under the user `logstash`. The `startup.options` file makes it easier for you to install multiple instances of the Logstash service. You can copy
+  the file and change the values for specific settings. Note that the `startup.options` file is not read at startup. If
+  you want to change the Logstash startup script (for example, to change the Logstash user or read from a different
+  configuration path), you must re-run the `system-install` script (as root) and pass in the new settings.
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: "and pass" should read "to pass"

**Comment by untergeek** [comment]
Other than the inline comments, this looks good!

File: docs/static/setting-up-logstash.asciidoc [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+| settings
+  | Configuration files, including `logstash.yml`, `jvm.options`, and `startup.options`
+  | `/etc/logstash`
+  | `path.settings`
+
+| conf
+  | Logstash pipeline configuration files
+  | `/etc/logstash/conf.d`
+  | `path.conf`
```
Comment (untergeek) [inline docs/static/setting-up-logstash.asciidoc]: My bad.  This should be `path.config` as the defaults in `logstash.yml` make plain:
```
# ------------ Pipeline Configuration Settings --------------
#
# Where to fetch the pipeline configuration for the main pipeline
#
# path.config:
```

**Comment by dedemorton** [comment]
@untergeek @suyograo I think I've addressed all concerns. Can I squash the commits and merge?

One last thing, tho: regarding the TOC, I think it makes sense to move the topics about the CLI and settings file to the new section. Here's what it would look like:

settinguptoc

Let me know what you think.

**Comment by suyograo** [comment]
@dedemorton LGTM from my side..

**Comment by suyograo** [comment]
+1 on moving to a new section

**Pull request closed without merging**